SPECS/pcre/pcre.spec

```
@@ -53,9 +58,9 @@ make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-mv -v %{buildroot}/usr/lib/libpcre.so.* %{buildroot}/lib &&
-ln -sfv ../../lib/$(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}%{_lib}/libpcre.so
-ln -sfv $(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}%{_lib}/libpcre.so.0
+mv -v %{buildroot}%{_libdir}/libpcre.so.* %{buildroot}/lib &&
+ln -sfv ../../lib/$(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_lib}/libpcre.so
+ln -sfv $(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_lib}/libpcre.so.0
```

Sorry, I missed the existing `%{_lib}` macros:

```suggestion
mv -v %{buildroot}%{_libdir}/libpcre.so.* %{buildroot}/lib &&
ln -sfv ../../lib/$(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_libdir}/libpcre.so
ln -sfv $(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_libdir}/libpcre.so.0
```